tests/assembly-llvm/x86_64-fortanix-unknown-sgx-lvi-generic-load.rs

```diff
@@ -1,17 +1,23 @@
 // Test LVI load hardening on SGX enclave code
 
+//@ add-core-stubs
 //@ assembly-output: emit-asm
-//@ compile-flags: --crate-type staticlib
-//@ only-x86_64-fortanix-unknown-sgx
+//@ compile-flags: --target x86_64-fortanix-unknown-sgx -Copt-level=0
+//@ needs-llvm-components: x86
+
+#![feature(no_core, lang_items, f16)]
+#![crate_type = "lib"]
+#![no_core]
+
+extern crate minicore;
+use minicore::*;
 
 #[no_mangle]
-pub extern "C" fn plus_one(r: &mut u64) {
-    *r = *r + 1;
+pub extern "C" fn load(a: &mut u64) {
+    *a = *a;
 }
```

Comment:
with `minicore`, we can't actually use the `+` operator. So hopefully this still tests what is relevant?